ci_framework/roles/operator_build/tasks/zuul_info.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Populate operators from zuul info
+  block:
+    - name: Get the PR owner using github api
+      register: pr_owner
+      ansible.builtin.shell: |
+        curl -s  -X GET -H \
+          -H "Accept: application/vnd.github+json" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/{{ item.project.name }}/pulls/{{ item.change }} | \
+          jq -r  '.head.repo.full_name'
+
+    - name: Add operator info with new content
+      ansible.builtin.set_fact:
+        zuul_info_operators: "{{ zuul_info_operators|default([]) + operator_info }}"
+      vars:
+        operator_info:
+          - name: "{{ item.project.short_name }}"
+            src: "{{ ansible_user_dir }}/{{ item.project.src_dir }}"
+            pr_sha: "{{ item.patchset }}"
+            pr_owner: "{{ pr_owner.stdout }}"
+  when:
+    - "'change_url' in item"
+    - "'operator' in item.project.short_name"
```

Comment:
In end-to-end job, we are calling running [ansible-playbook](https://github.com/openstack-k8s-operators/ci-framework/blob/main/.zuul.d/e2e-run.yml#L6) and the zuul related vars are not getting passed to nested ansible playbook that's why we are not above to discover it.

Reply:
@viroel https://github.com/openstack-k8s-operators/ci-framework/pull/136 will make zuul vars available to the nested ansible playbooks calls.

Reply:
if there's a better way to run the end-to-end, I'd be happy to learn :)

Reply:
thanks for fixing this